arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListBySubscription",
+        "operationId": "Namespaces_List",
```

Comment:
@sazeesha we will take these changes, when you release next .NET nuget lib make sure that version is updated as this is a breaking change.